common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1423,7 +1423,7 @@ def save_problem(self, data):
         self.track_function_unmask('save_problem_success', event_info)
         msg = _("Your answers have been saved.")
         if not self.max_attempts == 0:
-            msg = _("Your answers have been saved but not graded. Click 'Check' to grade them.")
+            msg = _("Your answers have been saved but not graded. Click '" + self.check_button_name() +  "' to grade them.")
```

Comment:
Also please verify that the strings for `check_button_name` are properly internationalized. There is a section on coverage testing in the above linked doc; if you do the coverage testing, that will be a way to test that both "check" and "final check" are properly internationalized.